### PR TITLE
LRDOCS-7579 Roles and Permissions Wordsmithed

### PR DESCRIPTION
**Pull request by sez11a**
Docs for defining roles and permissions. @rbohl

https://issues.liferay.com/browse/LRDOCS-7579

### CODE REVIEW

File: docs/dxp/7.x/en/users-and-permissions/roles-and-permissions/understanding-roles-and-permissions.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+1. Determine the function needed: _Manage Message Boards in Liferay DXP._
+1. Determine the scope: _Throughout the Virtual Instance (globally)._
+1. Create the properly scoped Role, assigning the permissions that power the function: _Message Boards Admin (Regular Role)._
+1. Create the User collection: _a User Group called Message Boards Administrators._
+1. Assign the Role to the User Collection: _Message Boards Administrators &rarr; Message Boards Admin Role._
+1. Assign Users to the User Collection: _Jane Slaughter &rarr; Message Boards Administrators._
+
+The conceptual diagram below shows how the User inherits the necessary permissions in Liferay DXP's Roles and Permissions system. Only one fully-wired Role assignment is displayed here (for the User who becomes a Message Boards Administrator), but you can see how drawing new arrows would link the User with permissions via other User collections and Role assignments.
+
+![Roles exist to link permissions efficiently with Users.](./understanding-roles-and-permissions/images/02.png)
```
Comment (jrhoun) [inline docs/dxp/7.x/en/users-and-permissions/roles-and-permissions/understanding-roles-and-permissions.md:32]: Can we create a design ticket to replace this diagram in the future? it's ok for now.

File: docs/dxp/7.x/en/users-and-permissions/roles-and-permissions/defining-role-permissions.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+1. Under the *General Permissions* heading, flag *Access in Control Panel* and *View*. This lets user group managers access the User Groups Control Panel portlet and view existing User Groups.
+
+1. Since you want to enable User Group managers to view User Groups and assign members to them, also check the *Assign Members* and *View* permissions under the *Resource Permissions* &rarr; *User Group* heading.
+
+1. Click *Save*.
+
+![Users assigned to the User Group Manager Role can Add available Users to any User Group, as well as remove them.](./defining-role-permissions/images/02.png)
+
+Now the User Group Manager Role has all the permissions necessary for adding Users to User Groups. User Group managers can view User Groups, assign members, and access User Groups in the Control Panel. The permission to view Users in the Control Panel was necessary because you must view Users to assign them as members of a Role. Without this permission, User Group managers see an empty list if they try to add Users to a Role.
+
```
Comment (jrhoun) [inline docs/dxp/7.x/en/users-and-permissions/roles-and-permissions/defining-role-permissions.md:54]: This section initially read as a non sequitur from the previous bit. We were talking about how to define permissions for a role. Now we're talking about defining (i think) a particular type of role - is the objective to provide a more practical permission defining use case/example?

Comment (rbohl) [reply docs/dxp/7.x/en/users-and-permissions/roles-and-permissions/defining-role-permissions.md:54]: Yes, this is the concrete example. I'd think  a transition sentence is all that's needed, after the note:

> Consider a Role for managing User Groups throughout the Virtual Instance.

Or something like that.

Comment (jrhoun) [reply docs/dxp/7.x/en/users-and-permissions/roles-and-permissions/defining-role-permissions.md:54]: Ok. thanks for confirming @rbohl - I wanted to make sure before I added in something.

**Comment by jrhoun** [comment]
Just pushed this up. Thanks guys.

Notable changes:

I removed the 'Advanced Roles and Permissions' section and moved "Assigning Roles to User Segments" down one level, so now users can actually find it.

I also removed some verbiage that came across a bit defensively here: https://github.com/liferay/liferay-learn/commit/f889a055d81d3074cdbba25eab82f74f96d0d6aa#diff-1fb21fc424c6fdb17cb0da05747fed04L3-L6

If you guys think it's necessary we can re-add it back - I just felt the tone was a unnecessarily defensive - and I tried to compensate for removing those lines by adding or reordering lines to direct people to where they needed to go.

**Pull request closed without merging**